react_week1/src/todo.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Todo = (props) =>{
```

Comment:
When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const Todo = ({description, deadline}) => { }

Then you can access the variables without using `props.`

Reply:
Hi @kasper, Thank you for your all valuable comments.I will rectify as soon as possible. :)